central/policy/datastore/datastore_impl.go

```diff
@@ -267,7 +267,7 @@ func (ds *datastoreImpl) UpdatePolicy(ctx context.Context, policy *storage.Polic
 
 	ds.policyMutex.Lock()
 	defer ds.policyMutex.Unlock()
-	// if feature flag turned on, check if categories need to be created/new policy category edges need to be created/
+	// Check if categories need to be created/new policy category edges need to be created/
```

Comment:
I'm confused by the old comment, because I don't see how this check works.

(Is the error check a short-circuit, in that the call will fail if the feature is not available?)

Reply:
The feature flag was removed, since feature was turned on by default, hence change in text.